src/main/java/io/github/thebusybiscuit/slimefun4/api/recipes/RecipeType.java

```diff
@@ -124,8 +124,8 @@ public void register(ItemStack[] recipe, ItemStack result) {
         } else {
             SlimefunItem slimefunItem = SlimefunItem.getById(this.machine);
 
-            if (slimefunItem instanceof MultiBlockMachine) {
-                ((MultiBlockMachine) slimefunItem).addRecipe(recipe, result);
+            if (slimefunItem instanceof MultiBlockMachine multiBlockMachine) {
+                multiBlockMachine.addRecipe(recipe, result);
```

Comment:
We do not always have to be so literal about it.
```suggestion
            if (slimefunItem instanceof MultiBlockMachine mbm) {
                mbm.addRecipe(recipe, result);
```